cibuildwheel/options.py

```diff
@@ -718,6 +719,11 @@ def _compute_build_options(self, identifier: str | None) -> BuildOptions:
 
             test_command = self.reader.get("test-command", option_format=ListFormat(sep=" && "))
             before_test = self.reader.get("before-test", option_format=ListFormat(sep=" && "))
+            xbuild_tools = shlex.split(
+                self.reader.get(
+                    "xbuild-tools", option_format=ListFormat(sep=" ", quote=shlex.quote)
+                )
+            )
```

Comment:
One request - could we detect the `["\u0000"]` value here and turn it into a `None` before it hits the BuildOptions object? I think the sentinel is fine within the options machinery, but those option values get printed out and I wonder what bugs we might inadvertently activate in CI systems with NULLs in our output (basically anything going though a C string function would truncate it). Also it's nice to isolate the bodge to the smallest subsystem possible.

Reply:
Makes sense - I've made this change in the most recent update.